sections/04_study.md

```diff
@@ -36,9 +36,16 @@ from gene expression.  Those could be divided into separate sub-sections.*
+
+In order to computationally predict TFBSs on a DNA sequence, researchers initially used consensus sequences and position weight matrices to match against a test sequence [@tag:Stormo2000_dna]. Simple neural network classifiers were then proposed to differentiate positive and negative binding sites, but did not show significant improvements over the weight matrix matching methods [@tag:Horton1992_assessment]. Later, SVM techniques outperformed the generative methods by using k-mer features \cite{Ghandi2014_enhanced, Setty2015_seqgl}, but string kernel based SVM systems are limited by expensive computational cost proportional to the number of training and testing sequences. Most recently, [@tag:Alipanahi2015_predicting] showed that a convolutional neural network models could achieve state of the art results on the TFBS task, are scalable to a large number of genomic sequences. [@tag:Lanchantin2016_motif] introduced several new convolutional and recurrent neural network models for predicting TFBSs, but it remains unclear which neural architectures work best for all samples and TFs. While neural architectures are rapidly changing and producing better results, it is clear that deep learning can be efficiently and effectively used to do functional prediction on the genome given raw data. 
+
+While accurately predicting transcription factors computationally is useful, it is important to understand how these computational models make their predictions. To handle this, several papers have focused on understanding machine learning models [@tag:Alipanahi2015_predicting	@tag:Lanchantin2016_motif@tag:Shrikumar2016_blackbox]. [@tag:Alipanahi2015_predicting] was the first to introduce a visualization method for a deep learning model on the TFBS task, and they did so by visualizing the learned convolution filters which were informative for the model’s prediction of a specific sample. However, this approach was specific to visualizing certain samples fed through their particular model. [@tag:Lanchantin2016_motif] introduced a suite of state-of-the-art deep learning models and new visualizations techniques for a more in-depth analysis of TFBSs. Furthermore, [@tag:Shrikumar2016_blackbox] introduced an advanced visualization method and toolbox for analyzing possible TFBS sequences. 
```

Comment:
Yes you can see where the build is failing due to that misformated citation in the [Travis log](https://travis-ci.org/greenelab/deep-review/builds/205947379#L324)